dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/event/data/AbstractJdbcEventAnalyticsManager.java

```diff
@@ -897,12 +897,12 @@ else if ( header.getValueType() == ValueType.REFERENCE )
     }
 
     /**
-     * Return SQL string bnased on Items a its params
+     * Return SQL string based on Items and its params
```

Comment:
Regarding the Javadoc, I think we could also make clear that the method takes into consideration "dimensions" and "filters".